### DISCUSSION TITLE
Importing comments

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#16519] [How to] [answered]
Hi there!
As far as I remember it was possible to import comments in vtiger (I think 6.x) adding: index.php?module=ModComments  you could see normal module view with all necessary options
I have some thousands comments to import into yetiforce. What will be an option? Anyone deal with it here?
Thanks for help

**Comment by vovpff** [comment, marked as answer]
You can import comments by selfwritten php script.
